src/styles/index.scss

```diff
@@ -1,3 +1,112 @@
+@import "vars";
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: $text-second-color;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
Use variables for all main values

Reply:
I did this, but I think if we use it once, then there is no need to put it in a variable